ega-charts/localega/templates/lega-pv.yaml

```diff
@@ -8,14 +8,14 @@ metadata:
   labels:
     type: local-storage
 spec:
-  storageClassName: {{ .Values.persistence.storageClass | quote }} 
+  storageClassName: {{ .Values.persistence.storageClass | quote }}
   claimRef:
     namespace: {{ .Release.Namespace }}
     name: {{ template "localega.inbox.fullname" . }}
   capacity:
     storage: {{ .Values.postgres.persistence.storageSize | quote }}
   accessModes:
-    - ReadWriteMany
```

Comment:
How will this solve the fact that the inbox and ingest needs a shared filesystem?
Cinder volumes can only be attached to a single pod.

Reply:
I think that support for volume multi-attach has been recently implemented in `cinder`: https://docs.openstack.org/cinder/latest/admin/blockstorage-volume-multiattach.html

Reply:
@jbygdell Also, as long as inbox and ingest are collocated on the same node, ReadWriteOnce both containers will be able  to mount the volume and write to it